Panels/PianoRoll/Grid.gd

```diff
@@ -0,0 +1,33 @@
+extends Control
+
+var scroll = 0 setget set_scroll
+
+
+func _ready():
+	Global.connect("window_size_changed", self, "fix_cells")
+	fix_cells(OS.window_size)
+
+func fix_cells(_window_size):
+	update()
+
+func set_scroll(val):
+	scroll += val
+	update()
+
+func _draw():
+	for x in range(OS.window_size.x-300):
+		draw_line(Vector2(90+x*14, 0), Vector2(90+x*14, get_node("../../../").rect_size.y),
```

Comment:
Parent node paths aren't recommended, maybe there's a better way to do this? :thinking: